uaclient/cli.py

```diff
@@ -1992,7 +1992,7 @@ def main(sys_argv=None):
 
     log_level = cfg.log_level
     console_level = logging.DEBUG if args.debug else logging.INFO
-    setup_logging(console_level, log_level, cfg.log_file)
+    setup_logging(console_level, log_level)
```

Comment:
Removing this will result in `setup_logging` instantiating a new UAConfig object unnecessarily. Let's leave it as it is for now
```suggestion
    setup_logging(console_level, log_level, cfg.log_file)
```

Reply:
Oh yes, it is like saying we default to the cfg.log_file unless we are non-root, the condition in `setup_logging`.